src/Proto3/Suite/DotProto/Internal.hs

```diff
@@ -50,6 +49,7 @@ import           Turtle                    (ExitCode (..), FilePath, Text,
                                             (</>))
 import           Turtle.Format             ((%))
 import qualified Turtle.Format             as F
+import NeatInterpolation qualified as Neat
```

Comment:
Why did you change just this one import to use ImportQualifiedPost, but not the others?